### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by sdwheeler**
# PR Summary

Updated the 7.1 release notes to be more descriptive.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [x] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [ ] Preview content
- [ ] Version 7.1 content
- [ ] Version 7.0 content
- [ ] Version 5.1 content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

**Review by SteveL-MSFT — APPROVED** [review]

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md [hunk 1]
```diff
@@ -1,47 +1,35 @@
...
+[CHANGELOG](https://github.com/PowerShell/PowerShell/blob/master/CHANGELOG/7.1.md) in the GitHub
+repository.
 
-We also have community support for:
+## PSReadLine 2.1.0
 
-- Arch Linux
-- Raspbian Linux
-- Kali Linux
+PSReadLine 2.1.0, which includes PowerShell 7.1 also include PSReadLine 2.1.0. This version includes
```
Comment (chasewilson) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:27]: Might be just me, but this sentence is a little confusing?

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md [hunk 2]
```diff
@@ -74,167 +67,127 @@ While not officially supported, the community has also provided packages for
...
-- macOS: `/usr/local/microsoft/powershell/6` is replaced by `/usr/local/microsoft/powershell/7`
+The following experimental features were added in this release:
 
-> [!NOTE]
-> In Windows PowerShell, the executable to launch PowerShell is named `powershell.exe`. In version 6
-> and above, the executable is changed to support side-by-side execution. The new executable to
-> launch PowerShell 7.1 is `pwsh.exe`. Preview builds will remain in-place as `pwsh-preview` instead
-> of `pwsh` under the 7-preview directory.
+- [Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace](../learn/experimental-features.md#microsoftpowershellutilitypsmanagebreakpointsinrunspace)
+  - PowerShell 7.1 extends this experimental feature to add **Runspace** parameter to all
```
Comment (chasewilson) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:82]:
```suggestion
  - PowerShell 7.1 extends this experimental feature to add the **Runspace** parameter to all
```

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md [hunk 3]
```diff
@@ -74,167 +67,127 @@ While not officially supported, the community has also provided packages for
...
+  `*-PSBreakpoint` cmdlets. The **Runspace** parameter specifies a **Runspace** object to interact
+  with breakpoints in the specified runspace.
+
+- [PSNativePSPathResolution](../learn/experimental-features.md#psnativepspathresolution) - this
```
Comment (chasewilson) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:86]:
```suggestion
- [PSNativePSPathResolution](../learn/experimental-features.md#psnativepspathresolution) - This
```

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md [hunk 4]
```diff
@@ -74,167 +67,127 @@ While not officially supported, the community has also provided packages for
...
+  feature allows you to pass PowerShell provider paths to native commands that don't support
+  PowerShell path syntax.
+
+- [PSCultureInvariantReplaceOperator](../learn/experimental-features.md#pscultureinvariantreplaceoperator) -
+  When the left-hand operand in a `-replace` operator statement is not a string, that operand is
+  converted to a string. With the feature enabled, the conversion does not use Culture settings for
+  string conversion.
+
+- [PSSubsystemPluginModel](../learn/experimental-features.md#pssubsystempluginmodel) lays the
+  groundwork to support future Predictive IntelliSense plug-ins
```
Comment (chasewilson) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:96]:
```suggestion
  groundwork to support future Predictive IntelliSense plug-ins.
```

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md [hunk 5]
```diff
@@ -74,167 +67,127 @@ While not officially supported, the community has also provided packages for
...
 
 ## Breaking Changes and Improvements
 
-For the latest information on changes and improvements, see the
-[CHANGELOG](https://github.com/PowerShell/PowerShell/tree/master/CHANGELOG) in the GitHub
-repository.
+- Fix `$?` to not be `$false` when native command writes to `stderr`
+  ([#13395](https://github.com/PowerShell/PowerShell/pull/13395))
+
+  It is common for native commands to write to stderr without intending to indicate a failure. With
```
Comment (chasewilson) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:103]:
```suggestion
  It is common for native commands to write to `stderr` without intending to indicate a failure. With
```

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md [hunk 6]
```diff
@@ -74,167 +67,127 @@ While not officially supported, the community has also provided packages for
...
+  this change `$?` is set to `$false` only when the native command also has a non-zero exit code.
+  This change is unrelated to the experimental feature `PSNotApplyErrorActionToStderr`.
 
-### Breaking Changes
+- Make `$ErrorActionPreference` not affect `stderr` output of native commands
+  ([#13361](https://github.com/PowerShell/PowerShell/pull/13361))
 
-<!-- TODO: Add descriptions for each breaking change - this might need to be a separate article that we link to -->
+  It is common for native commands to write to stderr without intending to indicate a failure. With
```
Comment (chasewilson) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:110]:
```suggestion
  It is common for native commands to write to `stderr` without intending to indicate a failure. With
```

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md [hunk 7]
```diff
@@ -74,167 +67,127 @@ While not officially supported, the community has also provided packages for
...
+  this change, stderr output is still captured in **ErrorRecord** objects, but the runtime no longer
```
Comment (chasewilson) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:111]:
```suggestion
  this change, `stderr` output is still captured in **ErrorRecord** objects, but the runtime no longer
```

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md [hunk 8]
```diff
@@ -74,167 +67,127 @@ While not officially supported, the community has also provided packages for
...
+
 - On Windows, `Start-Process` creates a process environment with all the environment variables from
-  current session, using `-UseNewEnvironment` creates a new default process environment (#10830) (Thanks @iSazonov!)
-- Do not wrap return result to `PSObject` when converting ScriptBlock to delegate (#10619)
-- Use invariant culture string conversion for `-replace` operator (#10954) (Thanks @iSazonov!)
+  current session, using `-UseNewEnvironment` creates a new default process environment
+  ([#10830](https://github.com/PowerShell/PowerShell/pull/10830)) (Thanks @iSazonov!)
 
-### Experimental Features
+- Do not wrap return result to `PSObject` when converting ScriptBlock to delegate
```
Comment (chasewilson) [inline reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:182]:
```suggestion
- Do not wrap return result to `PSObject` when converting a `ScriptBlock` to delegate
```

**Review by chasewilson — CHANGES_REQUESTED** [review]

**Review by theJasonHelmick — APPROVED** [review]
I really like this improvement to the release notes.  Especially the Experimental and Improvement sections. Easy to skim and gain information.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md (hunk 1 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:27]: I thought I fixed this already.

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md (hunk 2 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:82]: Fixed

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md (hunk 3 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:86]: Fixed

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md (hunk 4 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:96]: Fixed

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md (hunk 5 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:103]: fixed

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md (hunk 6 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:110]: fixed

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md (hunk 7 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:111]: fixed

File: reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md (hunk 8 above)
Comment (sdwheeler) [reply reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md:182]: fixed

**Review by chasewilson — APPROVED** [review]

**Pull request merged**